Formula/qt@5.rb

```diff
@@ -118,18 +118,11 @@ class QtAT5 < Formula
 
   # Fix build for GCC 11
   patch do
-    url "https://invent.kde.org/qt/qt/qtbase/commit/92646a2e0a4264dc9da97bc5e884a0116c0a32fb.patch"
-    sha256 "bd1a637da853e841259a9824481fe05b9c01563cc4e077e95fa8e0a7a479c653"
+    url "https://invent.kde.org/qt/qt/qtbase/commit/b340ca80cf4f994c26f01b017b8ca395a79324da.patch"
```

Comment:
This is essentially the same patch. Any idea why it was not part of the new release?

Reply:
Worth noting Qt 5.15 has a 1 year open source delay, which is why the second patch has only a part of the release now.

Though this particular patch is even older.

Looks like the first half of the patch was reverted: https://invent.kde.org/qt/qt/qtbase/-/commit/24a9a4e225191aa0306404e28c4c224909163e6d
And the second half of the patch was fixed on GCC's side: https://gcc.gnu.org/bugzilla/show_bug.cgi?id=98646

So it seems like we might not need the patch at all?

Reply:
> This is essentially the same patch. Any idea why it was not part of the new release?

Because the old commit It's not part of any branch now.
It could be deleted/cleaned by gitlab at any time.
KDE rebase their patches at every release.

Reply:
I'm not convinced we should keep this patch given the `qsharedpointer_impl.h` changes were reverted on the KDE branch due to it breaking rather than fixing things. The `qxcbwindow.cpp` changes to my understanding only affected beta versions of GCC 11 which is surely not used anymore, but if I'm wrong then slim it down to just that.